fec/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 2.1.5 on 2019-02-11 12:49
+# Generated by Django 2.2.6 on 2019-10-27 11:26
```

Comment:
The migrations weren't working, too many things had changed. So I just started over. There's no unique data in the Confero DB, so it's safe to just drop the database and start over